# Request
---
# Response
geometry_msgs/Point[] waypoints